src/main/webapp/app/instructor-dashboard/instructor-dashboard.component.html

```diff
@@ -31,6 +31,14 @@ <h2>{{exercise.course.title}} - {{exercise.title}} <small>{{results.length}} res
             <button class="btn btn-primary btn-sm" (click)="exportResults()">
                 <i class="fa fa-download fa-fw"></i>&nbsp;Export Results
             </button>
+            <button *ngIf="exercise.type !== 'quiz'"
```

Comment:
we also have modeling exercises, I think we need to exclude them here as well, because the zip export makes only sense for programming-exercises. As we will add more exercises in the future, I suggest, we explicitly test for programming exercises here